addOns/client/src/main/java/org/zaproxy/addon/client/ClientIntegrationAPI.java

```diff
@@ -41,9 +41,11 @@ public class ClientIntegrationAPI extends ApiImplementor {
 
     private static final String ACTION_REPORT_OBJECT = "reportObject";
     private static final String ACTION_REPORT_EVENT = "reportEvent";
+    private static final String ACTION_REPORT_ZEST_SCRIPT = "reportZestScript";
```

Comment:
`reportZestStatement` would be more accurate.

Reply:
Browser extension is sending by this name. So can only be changed when we will release new version of browser extension

Reply:
Something for next release then.

Reply:
We can still fix them anyway, the client add-on was not yet released. Fine if that's done in another PR since the browser extension will also have to change.

Reply:
Let's first make those changes in the extension and get it published again. Then I will create a new PR for this.